src/ui/pages/History/ActionItem/ActionItem.tsx

```diff
@@ -75,7 +81,9 @@ function ActionTitle({
 
   return (
     <UIText kind="body/accent">
-      {explorerUrl ? (
+      {!actionTitle ? (
+        <DelayedRender delay={1000}>[Missing network data]</DelayedRender>
```

Comment:
should there be a placeholder so that the element height doesn't change after `delay`?